README.rst

```diff
@@ -26,8 +26,7 @@ Features
 Installation
 --------------------------------------
 
-django-scribbler requires Django >= 1.3 and Python >= 2.6. Starting with version v0.2
-there will be experimental Python 3 support (3.2+). Using Python 3 requires using Django >= 1.5.
+django-scribbler requires Django >= 1.8 and Python >= 2.7 or >= 3.3.
```

Comment:
More of a Django issue documented in their supported version. I'd rather not repeat it here.